lasagne/tests/layers/test_special.py

```diff
@@ -2,6 +2,7 @@
 import numpy as np
 import pytest
 import theano
+import lasagne
```

Comment:
I'd prefer `from lasagne.layers import InputLayer, standardize, get_output` down in `test_standardize()` instead of a global import. (But also just for consistency, because most test suites were written to avoid a global `lasagne` import -- I don't know who started it. A possible advantage is that it's easier to track which Lasagne components specific test suites depend on.)